src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,22 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
+     *
      * @param email - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService currentuser = new UserService();
+
     public boolean login(String email, String password) {
+        if (currentuser.findByEmail(email) != null) {
+            if (currentuser.findByEmail(email).getEmail() == email
+                    && currentuser.findByEmail(email).getPassword() == password) {
+                return true;
+            }
+        }
```

Comment:
At first save your user what you get from findByEmail method and reuse this and don`t compare String type by "=="